vumi/transports/smpp/config.py

```diff
@@ -37,6 +37,11 @@ class SmppTransportConfig(Transport.CONFIG_CLASS):
         'for matching submit_sm_resp and delivery report messages. Defaults '
         'to 1 week.',
         default=(60 * 60 * 24 * 7), static=True)
+    completed_multipart_info_expiry = ConfigInt(
+        'How long (in seconds) to keep multipart message info for completed '
+        'multipart messages around to avoid pending operations accidentally '
+        'recreating them without an expiry time. Defaults to 1 hour.',
```

Comment:
Those are operations like "update this key in the hash", which we can't add expiration to in a single operation. We aren't expecting any of those after the message processing completes, but we can't guarantee that we won't get any. In the common case, adding an extra Redis call to every multipart `submit_sm_resp` we process doesn't seem worth it.